app/pages/Gifts/BuildGiftForm.view/BuildGiftForm.view.tsx

```diff
@@ -90,13 +91,18 @@ const BuildGiftForm: FC<BuildGiftFormProps> = ({
 }: BuildGiftFormProps) => {
   const classes = useStyles();
   const { t } = useTranslation('BuildGiftForm');
+  const { tokenId } = useSelector((state: ReduxStoreState) => state);
 
   // TODO - consider adding minimum gift ~ 1 MOB
   // We'll use this array in prep for future patterns with multiple accounts
-  const mockMultipleAccounts: Array<Account> = [
+  const mockMultipleAccounts: Array<{
+    b58Code: string;
+    balance: bigint;
+    name: string | null;
```

Comment:
I see this pattern in a few definitions - are we distinguishing between `null` and `undefined` here? Otherwise the pattern of `name?: string` might be better used here

Reply:
I'm not making that distinction intentionally here, but the functions using these types (that existed before this PR) expect `| null` and not undefined